trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py

```diff
@@ -16,26 +16,55 @@
 # XDR decoding tool available at:
 #   https://www.stellar.org/laboratory/#xdr-viewer
 #
+# ## Test Info
+#
+# The default mnemonic generates the following Stellar keypair at path 44'/148'/0':
+#   GAK5MSF74TJW6GLM7NLTL76YZJKM2S4CGP3UH4REJHPHZ4YBZW2GSBPW
+#   SDE2YU4V2IYSJIUH7MONDYZTSSLDXV5QDEGUUOLCU4TK7CZWTAXZ5CEG
+#
+# ### Testing a new Operation
+#
+# 1. Start at the Stellar transaction builder: https://www.stellar.org/laboratory/#txbuilder?network=test
+#   (Verify that the "test" network is active in the upper right)
+#
+# 2. Fill out the fields at the top as follows:
+#   Source account: GAK5MSF74TJW6GLM7NLTL76YZJKM2S4CGP3UH4REJHPHZ4YBZW2GSBPW
+#   Transaction sequence number: 4294967296 (see _create_msg)
+#   Base fee: 100
+#   Memo: None
+#   Time Bounds: <leave blank>
+#
+# 3. Select the operation to test, such as Create Account
+#
+# 4. Fill out the fields for the operation
+#
+# 5. Scroll down to the bottom of the page and click "Sign in Transaction Signer"
+#
+# 6. In the first "Add Signer" text box enter the secret key: SDE2YU4V2IYSJIUH7MONDYZTSSLDXV5QDEGUUOLCU4TK7CZWTAXZ5CEG
+#
+# 7. Scroll down to the bottom and look at the "signatures" section. The Trezor should generate the same signature
+#
 
 from base64 import b64encode
 from .common import TrezorTest
 from .conftest import TREZOR_VERSION
 from binascii import hexlify, unhexlify
 from trezorlib import messages as proto
 from trezorlib import stellar
-from trezorlib.tools import parse_path
 import pytest
 
 
 @pytest.mark.stellar
 @pytest.mark.xfail(TREZOR_VERSION == 2, reason="T2 support is not yet finished")
 class TestMsgStellarSignTransaction(TrezorTest):
 
-    ADDRESS_N = parse_path(stellar.DEFAULT_BIP32_PATH)
```

Comment:
please re-merge these changes I made in the meantime.
(that is: use `parse_path` instead of `expand_path`, use static variable instead of method)